tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md

```diff
@@ -0,0 +1,69 @@
+---
+author_name: Christian Savchev
+author_profile: https://github.com/christiansavchev  
+title: Create an Action of a Type Email
+description: Learn how to create an action of a type email
+keywords: tutorial
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product-function>sap-btp-cockpit ]
+primary_tag: software-product>sap-btp--cloud-foundry-environment
+parser: v2
+---
+
+# Create an Action of a Type Email
+<!-- description --> Learn how to create an action of a type email.
+
+## Prerequisites
+- You have activated Alert Notification in a trial account as described in [Set Up SAP Alert Notification service for SAP BTP for Initial Use](set-up-alert-notification).
+
+## You will learn
+- How to create an action of a type email, so that you can receive notifications to a given email. 
+    > If you want to receive notifications to other delivery channels, see the SAP Cloud Platform Alert Notification documentation at [Managing Actions](https://help.sap.com/docs/ALERT_NOTIFICATION/5967a369d4b74f7a9c2b91f5df8e6ab6/8a7e092eebc74b3ea01d506265e8c8f8.html).
+
+## Intro
+
+### Create an action of a type email.
+
+1. Open the **Actions** menu of the Alert Notification cockpit page.
+
+    ![Action](1-Action.png)
+
+2. Click on the **Create** button to open the dialog box. 
+
+    ![CreateAnAction](2-CreateAnAction.png)
+
+3. In the **Actions** dialog box, choose the prefered email action type and then click on the **Next** button.
+
+    ![DialogBox](3-DialogBox.png)
+
+4. Enter a unique name for your action in the **Name** field.
+
+    ![Name](4-Name.png)
+
+5. In the **Email Address** field, enter the destination email to which the notification must be sent and optionally provide a custom template for customizing your notification email. Then click on the **Create** button.
```

Comment:
...destination email address...

Reply:
Michael, one would not be able to tell from the md file if there is a validation as in v2, the vr file contains the reference to the step in the md file.  There might not be any indication of a validation in the md file and that is okay.

Reply:
You are right. I still have to get used to this part ;-)